tests/test_container.py

```diff
@@ -88,6 +88,48 @@ class Something(object):
     Something([1, 2, 3, 4])
 
 
+@pytest.mark.parametrize('test_type, correct, wrongs', [
+    (str, 'foo', [42, True]),
+    (int, 42, ['foo', ()]),
+])
+def test_typing_newtype_single(test_type, correct, wrongs):
```

Comment:
Can you split this test in two ? This test is testing multiple things at the same time: 

1. It tests it doesn't raise for a particular set of arguments
2. it tests it does raise for another particular set of arguments.

I'd prefer to have a clear defined scope for the test, without overlapping testing, in order to debug/understand and maintain this easier.

Eg:
`test_new_type_not_raises_when_correct_type`
`test_new_type_raises_with_incorrect_type`

Also, since these tests are for the newly supported NewType, this should be in their own test file.

Reply:
done

Reply:
Did you see my comment about moving these into a different test file ?